tests/data/region_processing/dsd/variable/variables.yaml

```
@@ -7,3 +7,5 @@
 - Share|Coal:
     definition: Share of Coal in the total primary energy mix
     unit:
+- World Variable:
```

Global mean temperature (increase) is a variable that is typically only reported at the World level...
```suggestion
- Temperature|Mean:
```